### PR TITLE
Update forging mgmt guide - Closes #842

### LINKED ISSUE #842: Add forging mgmt guide
Add a guide about how to enable forging for a Lisk node, similar to the Lisk Core forging guide. 

See: https://lisk.io/documentation/lisk-core/v3/management/forging.html

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #842

### CODE REVIEW

File: modules/ROOT/pages/guides/node-management/forging.adoc [hunk 1]
```diff
@@ -68,175 +91,123 @@ Please re-enter your password: ***
 ====
 Be sure to use a strong password.
 
-See the {url_wikipedia_password_strength}[Guidelines for password strength^] at Wikipedia for reference. 
+See the {url_wikipedia_password_strength}[Guidelines for password strength^] at Wikipedia for reference.
 ====
 
-* Type in the passphrase followed by the password required for encryption.
-* This will result in the creation of an `encryptedPassphrase` key-value pair.
+The hash onion can be generated with Lisk Commander the following way:
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/forging.adoc]:
```suggestion
The hash onion can be generated with Lisk Commander in the following manner:
```

File: modules/ROOT/pages/guides/node-management/forging.adoc [hunk 2]
```diff
@@ -68,175 +91,123 @@ Please re-enter your password: ***
...
-//For more information about the configuration of the Lisk SDK check out the xref:{url_guides_config}[configuration guide].
+For more information about the configuration of the Lisk SDK check out the xref:{url_guides_config}[configuration guide].
 
 [[forging_enable_disable]]
 == Enable/Disable Forging
 
 [CAUTION]
 ====
-Do not activate forging for the same delegate on multiple nodes.
+. Do not activate forging for the same delegate on multiple nodes at rthe same time.
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/forging.adoc]:
```suggestion
. Do not activate forging for the same delegate on multiple nodes at the same time.
```

File: modules/ROOT/pages/guides/node-management/forging.adoc [hunk 3]
```diff
@@ -68,175 +91,123 @@ Please re-enter your password: ***
...
+<5> Not required, when enabling forging for a delegate for the first time.
+Height of the previously prevoted block by any delegate.
+Must match the value in the `forger_info` data.
+<6> Not required, when enabling forging for a delegate for the first time.
+Height of the previously last forged block.
+Must match the value in the `forger_info` data.
+<7> Optional: If true, overrides `maxHeightPreviouslyForged` and `maxHeightPrevoted` values in the forger_info data.
 
-Use the following curl command to *disable the forging* for your delegate:
+Invoke the following action to *disale the forging* for a delegate:
```
Comment (chris529) [inline modules/ROOT/pages/guides/node-management/forging.adoc]:
```suggestion
Invoke the following action to *disable the forging* for a delegate:
```

**Review by chris529 — COMMENTED** [review]
Looks good, just a few minor points 👍

**Review by chris529 — APPROVED** [review]
LGTM!

File: modules/ROOT/pages/guides/node-management/forging.adoc [hunk 4]
```diff
@@ -37,6 +60,7 @@ To enable your node to forge for a particular delegate, firstly it is required t
 
 * `publicKey`: The publicKey of the delegate.
 * `encryptedPassphrase`: The symmetrically enrypted 12 word mnemonic passphrase of the delegate account.
+* `hashOnion`: The hash onion stores the random seeds of the delegate for each forging round.
```
Comment (ManuGowda) [inline modules/ROOT/pages/guides/node-management/forging.adoc:63]:
```suggestion
* `hashOnion`: The hash onion stores the random seeds to be used by delegate for each forging round.
```

File: modules/ROOT/pages/guides/node-management/forging.adoc [hunk 5]
```diff
@@ -68,175 +91,122 @@ Please re-enter your password: ***
...
-  "data": [
-    {
-      "forging": true,
-      "publicKey": "9bc945f92141d5e11e97274c275d127dc7656dda5c8fcbf1df7d44827a732664"
-    }
-  ],
-  "links": {}
-}
-----
+//@TODO: Add guide how to enable forging on another node, once app bootstrapping is added
```
Comment (ManuGowda) [inline modules/ROOT/pages/guides/node-management/forging.adoc:211]: todo is required?

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**